overlays/syncuser/rootfs/etc/passwd.ww

```diff
@@ -1,4 +1,8 @@
 # Uncomment the following line to enable passwordless root login
 # root::0:0:root:/root:/bin/bash
```

Comment:
We should remove this completely or make it in such a way that this root entry also runs through the UniqueField pipeline. You put so much effort on this, just to end up with two root entries.

Reply:
Thanks for calling this out. I refactored how `/etc/passwd` is building its output and moved the passwordless root login option to an optional inclusion managed by specifying a `PasswordlessRoot` tag. I think this is a real improvement, too, since it doesn't require modification of the template _and_ doesn't get forgotten in the template after debugging.

Reply:
(I also added a test for this case.)